src/content/docs/tutorials/bookesque/ch1.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Chapter 1
+tableOfContents: true
+---
+
+Welcome the world of Ratatui! This chapter will teach you what Ratatui is, and the core fundamentals
+to building your first TUI. In essence, there are three broad parts of a Ratatui application:
+
+1. UI rendering loop (rendering your TUI to the screen)
+2. Event Management (Keypresses, mouse click, etc.)
+3. State Management (Making your app dynamic)
+
+For all these options, Ratatui is incredibly flexible, allowing you to employ multiple approaches to
+tackle these parts. But let's not get ahead of ourselves. The rabbit hole of architectures and
+approaches is deep, and we'll explore it later on in this series, but to do that, we need to know
+how a Ratatui TUI runs.
+
+```svgbob
+          +------------+
+          | App starts |
+          +------+-----+
+                 |
+        +--------v---------+
+        |Initial UI Renders|
+        +--------+---------+
+                 |
+ +---------------v---------------+
+ |       +----------------+      |
+ |       | Event Received |<--+  |
+ |       +-------+--------+   |  |
+ |               |            |  |
+ |               |            |  |
+ |      +--------v---------+  |  |
+++-quit-+App Handles Events|  |  |
+||      +--------+---------+  |  |
+||               |            |  |
+||               |            |  |
+||        +------v-------+    |  |
+||        |Renders output+----+  |
+||        +--------------+       |
+|+-------------------------------+
+|
++----------------+
+                 |
+       +---------v----------+
+       |App Cleanup and Exit|
+       +--------------------+
+```
+
+## App Flow
+
+The Application flow can be separated into a startup (done using `ratatui::init()`), an application
+loop consisting of event handling and rendering, and a cleanup to restore the user's terminal back
+to its original state (done using `ratatui::restore`). The startup and cleanup are relatively
+simple, so we'll start with those first.
+
+### Startup and Cleanup
+
+#### Startup
+
+Setting up the terminal and necessary panic hooks for the application (we'll talk about why we need
+to later) can be done with a simple one liner:
+
+```rust
+let terminal = ratatui::init();
+```
+
+Under the hood, the `init()` function does the following:
+
+1. Sets the panic hook. A _panic hook_ is a custom function that is run automatically when a thread
```

Comment:
We should link to the available docs pages for stuff like these. e.g. [panic hook] [Alternate screen] etc.